### ISSUE TITLE
more scrape options like enableContentGrabber

### ISSUE THREAD

**Issue by dspstudio** [fguillot/picoFeed#92]
I'm wondering how can i retrieve the keywords from the scraped page when using enableContentGrabber().
I searched the code and couldn't find a way to implement it.

**Comment by mkresin** [comment]
Are you talking about the page-content after scraping a webpage?

**Comment by dspstudio** [comment]
Yes, after the page is scraped I wish to retrieve the meta keywords or other tags using xpath. Something like extending the rules to retrieve other parts of the page

**Comment by mkresin** [comment]
As far as I can see, Grabber::download() returns the raw html. There you go!

**Comment by Raydiation** [comment]
TL;DR: not possible without code modification.

I think the issue he has is related to the built in content grabbing. You define the stuff that should be extracted from the page as an xpath rule in the body segment like https://github.com/fguillot/picoFeed/blob/master/lib/PicoFeed/Rules/.blog.lemonde.fr.php#L4

If you dont match the meta keywords tags they are thrown away. So you need to change all rules and then modify the grabber output by removing the meta tags which is tedious. 

I dont really see a usecase in parsing the keywords and youd need to rewrite the whole content grabbing stuff to extract values, then define a custom result object which holds the content and the variables etc, etc, I think you see where this is going ;)

**Comment by mkresin** [comment]
For the sake of completeness: The head tags including any childes are stripped before the content is parsed.

@Raydiation extra punkte für deine Glaskugeling skills

**Comment by Raydiation** [comment]
:)

**Comment by dspstudio** [comment]
Thank you guys. I decided to use simple xml with xpath because its to complicated to modify the picofeed for my use case.
